Composer/packages/server/src/controllers/status.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { BackgroundProcessManager } from '../services/backgroundProcessManager';
+
+const getStatus = async (req, res) => {
+  const jobId = req.params.jobId;
+  try {
+    if (jobId) {
```

Comment:
if there is no jobId provided server should throw an error (400 bad request), if there is a job id provided but it doesn't exist 404.